msteams-platform/toolkit/configure-bot-capability.md

```diff
@@ -0,0 +1,662 @@
+---
+title: Configure bot capability
+author: surbhigupta
+description: Learn how to configure bot capability within Teams app.
+ms.author: v-preethah
+ms.localizationpriority: medium
+ms.topic: overview
+ms.date: 02/16/2024
+---
+
+# Configure bot capability within Teams app
+
+A bot, also known as a chatbot or conversational bot, is an app that responds to straightforward commands in a chat setting, providing relevant responses. Bots are commonly used for various tasks, such as notifying about build failures, supplying weather updates, bus schedules, or travel information. Bot interactions can range from simple question-and-answer exchanges to complex conversations. As a cloud application, a bot can offer secure and valuable access to cloud services and corporate resources. For more information, see [build bots for Microsoft Teams](../bots/what-are-bots.md).
+
+## Prerequisites
+
+To configure bot as an additional capability, ensure the following prerequisites are met:
+
+* The app manifest (previously called Teams app manifest) file.
+* A [Microsoft 365 account](../concepts/build-and-test/prepare-your-o365-tenant.md) to test the application.
+* [Microsoft Azure Storage account](/azure/storage/common/storage-account-create).
```

Comment:
An Azure account, not An Azure storage account.

Reply:
Updated.